src/snyk/common/watchers/settingsWatcher.ts

```diff
@@ -65,6 +68,12 @@ class SettingsWatcher implements IWatcher {
         }
       },
     );
+
+    SecretStorageAdapter.instance.onDidChange(event => {
```

Comment:
Should we have a separate `.ts` for it, since this is not really part of the settings watcher?
Other (maybe better idea): rename SettingsWatcher to ConfigurationWatcher :)